addons/web_tour/static/src/scss/tip.scss

```diff
@@ -63,7 +64,7 @@
     border: $o-tip-border-width solid $o-tip-color;
     border-radius: 50%;
     padding: 0; // ! Cannot be changed ! Javascript rely on this.
-    z-index: $zindex-modal + 1; // See comment on 'o_tooltip_parent' class
+    z-index: $zindex-modal + 2; // See comment on 'o_tooltip_parent' class
```

Comment:
what about using $zindex-tooltip instead ? https://getbootstrap.com/docs/4.0/layout/overview/#z-index
Seems appropriate as we want 'to position them above everything else', according to the comment above.
If we go for it, don't forget to adapt comments accordingly.

@qsm-odoo What do you think?

Reply:
Agreed :+1: `$zindex-tooltip` seems fine.

Reply:
thanks, done